qrpglesrc/newemp.pgm.sqlrpgle

```diff
@@ -45,7 +45,7 @@ dcl-s currentError like(XERR);
 autoEmpId = getNewEmpId();
 
 if (autoEmpId = '');
-  XERR = 'Unable to automatically generate an new ID.';
+  XERR = 'Something went really wrong.';
```

Comment:
The new message is far more generic.  The context does imply that it happened when generating a new ID.  So why generalize the message?